source/brailleDisplayDrivers/eurobraille.py

```diff
@@ -114,6 +116,9 @@
 })
 KEYS_ESITIME.update(KEYS_STICK)
 
+KEYS_BNOTE = KEYS_ESYS
+KEYS_BBOOK = KEYS_ESITIME
+
 DEVICE_TYPES={

```

Comment:
Can this get typing?

```suggestion
DEVICE_TYPES: Set[int, str] = {
```